src/core/socket.c

```diff
@@ -794,12 +794,11 @@ static void socket_close_fds(Socket *s) {
         assert(s);
 
         LIST_FOREACH(port, p, s->ports) {
+                bool rm;
```

Comment:
Nitpick: "rm" does not really mean "remove", just "was open" or something like that. That's a bit confusing when reading this code.

Also, I don't get the comment below. It says "not delete any sockets in the file system here", and removes SOCKET_FIFO a few lines later. If you understand what is going on here, could you update the comment too?